### PR TITLE
Update run

### PR DESCRIPTION
**Pull request by xytosis**
Updating the run script to look for 0.2.3-SNAPSHOT.jar
Updating the build.gradle file to remove useless lines.
Updated testsubscriber to wait longer

### CODE REVIEW

File: reactivesocket-tck-drivers/run.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-java -cp build/libs/reactivesocket-tck-drivers-0.2.2-SNAPSHOT.jar io.reactivesocket.tckdrivers.main.Main "$@"
+java -cp build/libs/reactivesocket-tck-drivers-0.2.3-SNAPSHOT.jar io.reactivesocket.tckdrivers.main.Main "$@"
```
Comment (yschimke) [inline reactivesocket-tck-drivers/run.sh]: Can you either make this find the right version automatically, or use something like this http://stackoverflow.com/a/21360609/1542667

Comment (xytosis) [inline reactivesocket-tck-drivers/run.sh]: That's definitely a more robust solution

**Comment by xytosis** [comment]
wait something broke... will reupdate

**Comment by xytosis** [comment]
Will merge these changes on other pull request https://github.com/ReactiveSocket/reactivesocket-java/pull/161

**Pull request closed without merging**